proposed/http-message.md

```diff
@@ -1100,7 +1106,8 @@ interface StreamableInterface
      *
      * @see seek()
      * @link http://www.php.net/manual/en/function.fseek.php
-     * @return bool Returns TRUE on success or FALSE on failure.
+     * @return bool Returns TRUE on success.
```

Comment:
We could remove the return value. Since you'd have to wrap it anyways in order to use it with StreamWrapper, it may not be necessary:

```php
public function stream_seek($offset, $whence = SEEK_SET)
{
    $this->seek($offset, $whence);
    return true;
}
```

I'm going to update these various cases.